jib-cli/src/integration-test/java/com/google/cloud/tools/jib/cli/JarCommandTest.java

```diff
@@ -331,4 +332,16 @@ private static String getContent(URL url) throws InterruptedException {
     }
     return null;
   }
+
+  private String getDockerHostForHttpRequest() {
```

Comment:
The `get` prefix is generally reserved for getters where we have one return statement. Since we have a little bit of logic here, what are your thoughts on this? 

```suggestion
  private String fetchDockerHostForHttpRequest() {
```